### DISCUSSION TITLE
how to seach a keyword in requestbody(jsonrequest or formdata)

### DISCUSSION THREAD

**Discussion by VinterHe** [Kong/insomnia#4260] [Ideas]
Currently, we can search keyword in title, but i want to seach a keyword in requestbody(jsonrequest or formdata) too, just like postman.

The feature  name in postman is "Find and Replace", and the location is in the left of bottom.

**Comment by wdawson** [comment]
Thanks for submitting this @VinterHe ! I'll move this to GitHub Discussions as an Idea so we can better keep track of this.
